hazelnut-cake.md

```diff
@@ -0,0 +1,29 @@
+# Hazelnut cake
+#### Recipe by Nicolai von Neudeck (self-employed) & Annette von Neudeck, née Matton
+Annette gifted Nicolai this hazelnut cake recipe in a newly hand written cookbook when he moved from the Netherlands to Berlin. While the new cookbook was lost in a flatshare in 2006, the recipe has been in the family for at least a hundred years and is from a time when gout was a more common disease. Hence the cake is much richer than most cakes today. It is easy to bake, the ingredients do the heavy lifting.
```

Comment:
Maybe we should take the hundred years out. It sounds pretentious and I think age is not a sign of quality for a recipe.